.github/workflows/test.yaml

```diff
@@ -1282,6 +1275,86 @@ jobs:
         if: failure()
         run: bash .github/slack/notify-failure.sh ${{ github.job }} ${{ matrix.combo }}
 
+  driver-adapters-wasm:
+    needs: [detect_jobs_to_run]
+    if: ${{ fromJson(needs.detect_jobs_to_run.outputs.jobs)['driver-adapters-wasm'] == true }}
+
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        combo:
+          - pg-node-basic-wasm
+          - pg-node-basic-wasm-edge-import
+          - pg-cf-basic
+          - neon-http-cf-basic
+          - neon-cf-basic
+          - planetscale-cf-basic
+          - turso-cf-basic
+        clientEngine: ['wasm']
+        os: [ubuntu-20.04]
+    runs-on: ${{ matrix.os }}
+    concurrency: ${{ github.job }}-${{ matrix.combo }}-${{ matrix.clientEngine }}
+    env:
+      NODE_ENV: development
```

Comment:
Is `NODE_ENV` really needed here?

Reply:
It's probably not used indeed

Reply:
We should remove all of them via a separate PR